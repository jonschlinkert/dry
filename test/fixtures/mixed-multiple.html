{% extends "multiple-mixed" %}
{% block "one" %}
one: "pages/multiple.html"
{% endblock %}
{% block replace="two" %}
two: "pages/multiple.html"
{% endblock %}
{% block append="three" %}
three: "pages/multiple.html"
{% endblock %}
{% block prepend="four" %}
four: "pages/multiple.html"
{% endblock %}
FIVE IS SKIPPED INTENTIONALLY
{% block append="six" %}
six: "pages/multiple.html"
{% endblock %}