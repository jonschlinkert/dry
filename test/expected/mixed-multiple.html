
head: "blocks/multiple-mixed.html"

one: "pages/multiple.html"

two: "pages/multiple.html"

three: "blocks/multiple-mixed.html"
three: "pages/multiple.html"

four: "pages/multiple.html"
four: "blocks/multiple-mixed.html"

five: "blocks/multiple-mixed.html"

six: "blocks/multiple-mixed.html"
six: "pages/multiple.html"

foot: "blocks/multiple-mixed.html"
